stylist/views.py

```diff
@@ -78,23 +55,7 @@ def get_context_data(self, *args, **kwargs):
         return context
 
     def form_valid(self, form):
-        custom_vars = build_scss(self, form.cleaned_data)
-        
-        instance = Style.objects.get(uuid=self.kwargs["uuid"])
-        filename = css_file_path(instance, instance.name + "_preview.css")
-
-        # if a preview is currently active, delete the old file
-        if self.request.session.get("preview_path"):
-            default_storage.delete(self.request.session["preview_path"])
-        content = sass.compile(filename=settings.STYLIST_SCSS_TEMPLATE, include_paths=[gettempdir()])
-        preview = default_storage.save(filename, ContentFile(content.encode()))
-
-        # bust cache to always use the most recent preview file
-        timestamp = "?" + str(int(datetime.utcnow().timestamp()))
-        self.request.session["preview_css"] = default_storage.url(preview) + timestamp
-        self.request.session["preview_path"] = preview
-        os.remove(custom_vars.name)
-        
+        self.request.session["preview_css"] = Style.objects.get(uuid=self.kwargs["uuid"]).attrs
```

Comment:
might need to grab the values that are being submitted (cleaned_data) vs the actual instance

Reply:
tested it out and this is definitely the case. works for previewing an existing theme but not for previewing unsaved changes